### ISSUE TITLE
Missing analytics data (starting December 1st-ish)?

### ISSUE THREAD

**Issue by GUI** [18F/api.data.gov#362]
I'm not totally certain about this, but wanted to add an issue to investigate further:

While looking at some analytics in the admin, I noticed there was a discernible drop-off in traffic starting around November 30 or December 1st. It seems like now we're averaging only 50-60% of the traffic we were seeing last week. I'm seeing a similar drop-off across all agencies, which seems rather peculiar since we don't usually see such uniform patterns across different agencies. On a quick look at the server logs, based just on the volume of the log files, things look much steadier than what we're seeing in the admin (in other words, the file size of the logs for last week and this week look similar, but the admin is reporting much lower traffic this week).

All of this leads me to believe something's maybe not operating correctly with our log data being sent to Elasticsearch. It almost seems as though 1 of our 2 servers isn't sending its log data to Elasticsearch. I still need to investigate further, though.

**Comment by cmc333333** [comment]
@GUI are you still also populating Kylin? Would that be downstream of these log files or could you use it as a point of reference?

**Comment by GUI** [comment]
Hm, good idea about Kylin, but I'm unfortunately not sure if that's still populating or aggregating. I'll have to take a closer look (and at some point actually wrap all that up...).

Based on a quick look, though, I think the issue might have been upstream of that. All our nginx log files on each server look normal, but the rsyslog files on 1 of the 2 servers was much smaller than the other. So it seems like possibly rsyslog on 1 of the 2 servers croaked (or nginx's ability to communicate with rsyslog), and therefore our log data wasn't being shipped by rsyslog to Elasticsearch (which would also mean they weren't getting shipped to Kylin).

I took the quick and easy way out this morning and just tried restarting API Umbrella on each server. It looks like that probably fixed things, since I restarted things around 9AM, and the log data appears to have immediately jumped back to normal levels since then:

<img width="1468" alt="screen shot 2016-12-06 at 7 45 09 pm" src="https://cloud.githubusercontent.com/assets/12112/20952989/9a26bc0a-bbed-11e6-9419-1bf3795933f5.png">

So I'll continue to keep my eye on things, but hopefully things are at least functioning normally now.

However, we still need to recover the missing log data since December 1 (we should be able to process the basic details from the nginx logs), which I'm hoping I might have a chance to do later this week. More importantly, we also need to figure out the underlying issue of what went wrong to prevent this from happening again. No obvious errors jumped out at me, but I haven't really had a chance to look at things in more detail.
